### ISSUE TITLE
Setup Layout/RescueEnsureAlignment cop 

### ISSUE THREAD

**Issue by roman-dubrovsky** [datarockets/datarockets-style#133]
Related with the next issue https://github.com/datarockets/datarockets-style/issues/130

Need to set up the rule for the next cop: `Layout/RescueEnsureAlignment`

Now It suggests correcting to 

```ruby
  def facebook_util
    return @facebook_util if defined? @facebook_util

    @facebook_util = begin
                       FacebookUtil.new(@account.id)
                     rescue
                       nil
                     end
  end
```

should be

```ruby
  def facebook_util
    return @facebook_util if defined? @facebook_util

    @facebook_util = begin
      FacebookUtil.new(@account.id)
    rescue
      nil
    end
  end
```

**Comment by lazycoder9** [comment]
@roman-dubrovsky after a little investigation found that related cops are buggy now ([Layout/EndAlignment](https://github.com/rubocop-hq/rubocop/issues/6774) and [Layout/RescueEnsureAlignment](https://github.com/rubocop-hq/rubocop/issues/6918)). I think we should postpone it until fixes will release

**Comment by roman-dubrovsky** [comment]
@lazycoder9 why is it buggy? can you write your thoughts what is wrong? or just links to issues or PRs?

**Comment by lazycoder9** [comment]
@roman-dubrovsky Oh, it seems the links in the previous comment were not obvious, but they were links to issue

**Comment by nikitasakov** [comment]
This issue still not resolved

**Comment by ianfixes** [comment]
I came here from #6918 ... I'm looking for guidance on whether to propose a test case for this or open a separate feature request entirely.

The code in question is:

```ruby
  private memoize def do_something
    "Something"
  rescue NameError
    nil
  end
```

Rubocop `0.77.0` wants this:

```ruby
  private memoize def do_something
    "Something"
                  rescue NameError
                    nil
  end
```

If I remove `memoize`, Rubocop acts sensibly and allows this:

```ruby
  private def do_something
    "Something"
  rescue NameError
    nil
  end
```

So, is a variation on what's listed in the issue description or should I open a new feature request for an explicit `EnforcedStyleAlignWith: start_of_line` configuration option for this cop?

**Comment by paydaylight** [comment]
Seems this fixed in [this](https://github.com/rubocop-hq/rubocop/issues/8710) issue and released in [0.91.1](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md#0911-2020-09-23).
